### ISSUE TITLE
Add pager to coude_output

### ISSUE THREAD

**Issue by vchuravy** [JuliaDebug/Cthulhu.jl#232]

**Comment by thazhemadam** [comment]
Ideally, I think only the actual traces themselves should be limited by the pager, and the menu would show up nonetheless. Would that be possible?

**Comment by thazhemadam** [comment]
Somewhat related - would it be a welcome change to colorize the headings for the menu options as well (I'm talking `Toggles`, `Show`, etc). It should be a relatively simple change and would make it much more easier to find the menu options 😅

**Comment by vchuravy** [comment]
Probably best to do that in two PRs and important to have a toggle to turn it off.

**Comment by thazhemadam** [comment]
> Probably best to do that in two PRs and important to have a toggle to turn it off.

Understood. I'll open a separate issue to track that.

**Comment by thazhemadam** [comment]
> Ideally, I think only the actual traces themselves should be limited by the pager, and the menu would show up nonetheless. Would that be possible?

As I see it, the issue with using TerminalPager would be if we just called `pager` on the string being printed to the buffer here, 
https://github.com/JuliaDebug/Cthulhu.jl/blob/6880b9e47f57625e890594e0d0e11c59393b4bb9/src/Cthulhu.jl#L301 it would automatically open as a pager. That basically means that users wouldn't be able to see the menu until they `q` from the pager, and once they see the menu, they won't be able to immediately scroll back to see the code they just scrolled past 😕

**Comment by vchuravy** [comment]
yeah that's not ideal. maybe we need a deeper integration between TerminalPager and TerminalMenu is needed
